src/elements/content-explorer/ContentExplorer.js

```diff
@@ -1277,8 +1277,26 @@ class ContentExplorer extends Component<Props, State> {
     handleSharedLinkSuccess = (newItem: BoxItem) => {
```

Comment:
This function seemed to be missing some functionality - the comments indicate it creates a shared link using enterprise defaults, but that part was implemented before